content/blog/2025/fundamentals_architecture.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "What is Software Architecture? (IMHO)"          # Name of the blog
+description: |
+  I believe everyone agrees with Ralph Johnson’s definition of software architecture: "Architecture is about the important stuff. Whatever that is". While I fully agree with this definition, I think adding more clarity would help better understand what it means.
+
+date: "2025-02-04"          # Data post is published
+image: ""          # Displayed when referenced in listing pages
+tags:            # Used for SEO optimization and browsing across the site.
+- "Software Architecture"
+- "Fundamentals"
+externalLink: ""      # Full URL to override listing links to an external page
+authors:
+- "ilya-hardzeenka.md"            # An array of authors of the post (filenames in content/authors).
+draft: false
+---
+
+**This blog is inspired by *[Fundamentals -> Understanding Architecture]({{% relref "/guide/fundamentals/understanding_architecture/" %}})* in our Guide.**
```

Comment:
Maybe replace "is inspired by" with "explores themes covered in" or "complements the information presented in" to seem a bit modest.